src/rp2_common/pico_cyw43_driver/btstack_hci_transport_cyw43.c

```diff
@@ -148,6 +151,13 @@ static void hci_transport_cyw43_process(void) {
         } else {
             has_work = false;
         }
+// PICO_CONFIG: PICO_BTSTACK_CYW43_MAX_HCI_PROCESS_LOOP_COUNT, max number of iterations of the hci processing loop, type=int, default=infinite, advanced=true, group=pico_btstack
```

Comment:
`tools/extract_configs.py` doesn't support `default=infinite`. Perhaps it's better to just remove the `default` and change the description to "limit the max number..." ? (which hopefully makes it obvious that if `PICO_BTSTACK_CYW43_MAX_HCI_PROCESS_LOOP_COUNT` isn't set, there's no limit?)

Reply:
waiting on this